frontend/src/containers/RegisterPage/RegisterPage.js

```diff
@@ -23,22 +26,26 @@ class RegisterPage extends React.Component {
     };
     register(data)
       .then(() => {
+        this.setState({ isLoading: false });
         const { history } = this.props;
         history.push('/confirm-email');
       })
       .catch((ex) => {
+        this.setState({ isLoading: false });
         if (ex.response === undefined) {
           this.setState({ showGeneralError: true });
           return;
         }
         const { status } = ex.response;
         if (status === 409) this.setState({ showUserExistsError: true });
         else this.setState({ showGeneralError: true });
+      })
+      .finally(() => {
       });
```

Comment:
Empty finally block

Reply:
It was left by me I think, but could you please delete it?

Reply:
CXDDDDDDDDDDDd